zstash/hpss.py

```diff
@@ -19,16 +19,14 @@ def hpss_transfer(hpss, file_path, transfer_type, cache, keep=None):
             # https://unix.stackexchange.com/questions/46915/get-the-chmod-numerical-value-for-a-file
             display_mode = "stat --format '%a' {}".format(file_path).split()
             output = subprocess.check_output(display_mode).strip()
-            # FIXME: On Python 3 '{}'.format(b'abc') produces "b'abc'", not 'abc'; use '{!r}'.format(b'abc') if this is desired behavior mypy(error)
-            logger.info("{} original mode={}".format(file_path, output))  # type: ignore
+            logger.info("{!r} original mode={!r}".format(file_path, output))
```

Comment:
`'{}'.format(b'abc')` and `'{!r}'.format(b'abc')` both return `"b'abc'"` for me. I suppose just adding the `!r` is fine?

Reply:
I think `mypy` is complaining to be explicit by adding `!r` if `"b'abc'"` is the expected output. Looks good to me